etc/tsansup.txt

```diff
@@ -13,22 +13,22 @@ race:BloombergLP::bdlma::ConcurrentPool::deleteObject
 # attempts to explain why there is no race.
 race:BloombergLP::bcema_Pool<*>::allocate
 
-# Not sure what the problem is here, but tsan can't show the other stack, so
-# there's nothing to look into
-race:__tsan_atomic32_fetch_add
```

Comment:
Removed because it seems [TSan issue](https://github.com/google/sanitizers/issues/953) and probably fixed. No failures were observed.